runtime/near-vm/Cargo.lock

```diff
@@ -0,0 +1,2205 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This file I think is no longer relevant, since the top-level Cargo.lock is used?